lib/Dancer2/Core/App.pm

```diff
@@ -833,6 +833,41 @@ sub log {
     $logger->$level(@_);
 }
 
+sub send_as {
+    my $self = shift;
+    my ( $type, $data, $options ) = @_;
+    $options ||= {};
+
+    # allow lower cased serializer names
+    my $prefix = 'Dancer2::Serializer::';
+    my $serializer_class;
+    for ( uc $type, $type ) {
+        if ( Class::Load::try_load_class( "$prefix$_" ) ) {
```

Comment:
`Class::Load::XS` is not used in any significant way by `try_load_class` - most of the lifting in there is via `Module::Runtime`. I've been using `Module::Runtime::use_module` with new projects for a while (wrapped in `Try::Tiny`) mostly for reduced deps but mostly for this:

```
my $obj = use_module('MyClass')->new(%args);
```
Sweet.